Live.Avalonia/LiveFileWatcher.cs

```diff
@@ -2,55 +2,53 @@
 using System.IO;
 using System.Reactive.Linq;
 using System.Reactive.Subjects;
+using System.Text.RegularExpressions;
 
 namespace Live.Avalonia
 {
     internal sealed class LiveFileWatcher : IDisposable
     {
-        private readonly Subject<string> _fileChanged = new Subject<string>();
-        private readonly Action<string> _logger;
-        
-        private IDisposable _timerSubscription;
-        private string _latestSignature;
+        readonly Subject<string> _fileChanged = new Subject<string>();
+        readonly Action<string> _logger;
+
+        FileSystemWatcher watcher = new FileSystemWatcher()
```

Comment:
Probably worth marking this field as `readonly`, as we aren't reassigning the watcher anywhere.